packages/client/lib/rpc/modules/eth.ts

```diff
@@ -957,6 +963,34 @@ export class Eth {
 
     return bufferToHex(tx.hash())
   }
+
```

Comment:
The other methods have some docs before them, perhaps we could add this here too:
```
  /**
   * Returns an object with data about the sync status or false.
   * @param params An array of three parameters:
   *   1. address of the account
   *   2. array of storage-keys which should be proofed and included
   *   3. integer block number, or the string "latest" or "earliest"
   * @returns An account object
   **/
   ```